news/671.feature

```diff
@@ -0,0 +1,7 @@
+Improved serialization/deserialization of the Plone Site root to make
+possible an opt-in to support Tiles attributes on it. The @workflow endpoint
+now answers an empty object instead of a 404 for the Plone Site Root object,
+hiding the implementation details (the Plone Site Root has no workflow for
+historical reasons) and equalizing it to the other content types, making the
+API more consistent.
+[sneridagh]
```

Comment:
I tried to improve the wording and I split up the entry. @sneridagh please double check if you are happy with my amendment.

Reply:
@tisto thank you! IMHO that summarizes the changes perfectly. From what I understand, the "opt-in" part just referred to the fact that that the tiles-specific fields aren't required in the deserializer, which is the case in most places in the API, so that can be omitted.

Reply:
@tisto Absolutelly fine with me!!